internal/app/plan.go

```diff
@@ -63,8 +65,17 @@ func (p *plan) addCommand(cmd command, priority int, r *release) {
 	p.Commands = append(p.Commands, oc)
 }
 
+// logDecision adds the decisions made to the plan.
+// Depending on the debug flag being set or not, it will either log the the decision to output or not.
+func (p *plan) logDecision(decision string, priority int, decisionType decisionType) {
+	p.addDecision(decision, priority, decisionType)
+	log.Debug(decision)
```

Comment:
same as the comment above .. this logs the plan decisions with DEBUG prefix when only `verbose` is used. In fact those decisions are logged anyway in the plan in all cases. So maybe this log can be deleted here.

Reply:
I've removed the whole function and use `addDecision` directly everywhere.